doc/source/misc/faq.rst

```diff
@@ -0,0 +1,24 @@
+Frequently Asked Questions
+==========================
+
+Star scoring
+-------------
+
+Rock Band style scoring
++++++++++++++++++++++++
```

Comment:
I've removed any distinction of different co-op game already so this documentation is a bit out of date iirc.

Reply:
Ok, I'll remove this part, then.